src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCell.swift

```diff
@@ -14,7 +14,7 @@ class HealthCertificateQRCodeCell: UITableViewCell, ReuseIdentifierProviding {
 		isAccessibilityElement = false
 
 		qrCodeView.isAccessibilityElement = true
-		titleLabel.isAccessibilityElement = true
+		titleLabel.isAccessibilityElement = false // https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9976
```

Comment:
Another technique that I often use when the elements are out of order is to set the `accessibilityElements` array in the correct order. Then the correct elements would be still highlighted when read out loud

Reply:
Yes I tried that but this is utterly complicated since this View as so many subviews for different use cases, that it seemed better to just take out this element completely. Since the recommendation from the a11y expert was to have it read it out as one element and not two.